docs/azure/sdk/logging.md

```diff
@@ -202,6 +195,34 @@ Since the `Logging:LogLevel:Azure.Core` key is set to `Debug`, Azure Core librar
 
 For more information, see [Logging in .NET Core and ASP.NET Core](/aspnet/core/fundamentals/logging/).
 
+## Log HTTP request and response bodies
+
+When troubleshooting unexpected behavior with a client library, it's helpful to inspect the following items:
+
+- The HTTP request body sent to the underlying Azure service's REST API.
+- The HTTP response body received from the Azure service's REST API.
+
+By default, logging of the aforementioned content is disabled. To enable logging of the HTTP request and response bodies, complete the following steps:
+
+1. Set the client options object's <xref:Azure.Core.DiagnosticsOptions.IsLoggingContentEnabled%2A> property to `true`. For example:
+
+    ```csharp
+    var options = new SecretClientOptions
+    {
+        Diagnostics = 
+        {
+            IsLoggingContentEnabled = true,
+        }
+    };
+    ```
+
+1. Set an [event level](/dotnet/api/system.diagnostics.tracing.eventlevel) of either `Verbose` or `LogAlways`. For example:
```

Comment:
This relates to the header about using built-in methods for logging. I wonder if we should keep this section neutral in terms of how users are doing the logging. Regardless of whether they use one of the built-in methods, do custom logging by creating their own AzureEventSourceListener instance, or use appsettings.json in an ASP.NET Core app, they still need to set IsLoggingContentEnabled to true, and they need to have a log level of verbose/debug or higher.